public/app/features/alerting/unified/components/silences/SilencesDrawer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { Drawer, Stack } from '@grafana/ui';
+import { AlertManagerPicker } from 'app/features/alerting/unified/components/AlertManagerPicker';
+import { GrafanaAlertmanagerDeliveryWarning } from 'app/features/alerting/unified/components/GrafanaAlertmanagerDeliveryWarning';
+import { SilencesEditor } from 'app/features/alerting/unified/components/silences/SilencesEditor';
+import { getDefaultSilenceFormValues } from 'app/features/alerting/unified/components/silences/utils';
+import { useAlertmanager } from 'app/features/alerting/unified/state/AlertmanagerContext';
+import { isGrafanaRulerRule } from 'app/features/alerting/unified/utils/rules';
+import { CombinedRule } from 'app/types/unified-alerting';
+
+type Props = {
+  rule: CombinedRule;
+  onClose: () => void;
+};
+
+/**
+ * For a given rule, renders a drawer containing silences editor and Alertmanager selection
+ */
+const SilencesDrawer = ({ rule, onClose }: Props) => {
```

Comment:
How about `SilenceRuleDrawer` to emphasise the component creates rule-bounded silences?